ui/tests/integration/components/kv/kv-data-fields-test.js

```diff
@@ -73,4 +73,18 @@ module('Integration | Component | kv-v2 | KvDataFields', function (hooks) {
     assert.dom(FORM.maskedValueInput()).hasValue('bar');
     assert.dom(FORM.dataInputLabel({ isJson: false })).hasText('Version data');
   });
+
+  test('it shows readonly info rows when viewing secret details of simple secret', async function (assert) {
```

Comment:
Can we add one more test with `type=details` and a complex secret, ensuring it shows the non-editable JSON editor?